modules/src/application/ics20_fungible_token_transfer/relay_application_logic/send_transfer.rs

```diff
@@ -34,6 +35,9 @@ where
             Kind::SequenceSendNotFound(msg.source_port.clone(), msg.source_channel.clone())
         })?;
 
+    let timeout_timestamp = Timestamp::from_nanoseconds(msg.timeout_timestamp)
+        .map_err(|_| Kind::InvalidPacketTimestamp)?;
```

Comment:
Can we store the invalid timestamp (`u64`) in the `InvalidPacketTimestamp` variant and show it in the error message? This might help debug some issues.